http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueries.scala

```diff
@@ -401,7 +401,14 @@ object RuntimeServiceDbQueries {
     val runtimeInOwnedProjects: Option[ClusterTable => Rep[Option[Boolean]]] =
       if (ownedProjects.isEmpty)
         None
-      else
+      else if (cloudContext.isDefined) {
+        // If cloudContext is defined, we're already applying the filter elsewhere.
```

Comment:
Could you link to where the filter is already applied?

Reply:
yep..it's just down below in the same method https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueries.scala#L432-L434